### PR TITLE
Fixed bug Cant get it to work

### PR DESCRIPTION
**Pull request by hendrathings**
Fixed issue #65 Cant get it to work and update some style

### CODE REVIEW

**Comment by Magiweb** [comment]
@hendrathings this is the bugfix right? But it's not pulled, and it looks like eXon has made other changes to his code in the mean time. @eXon, could we get this merged?

**Comment by hendrathings** [comment]
yes, @eXon please check and merge immediately

File: src/Vimeo.js [hunk 1]
```diff
@@ -67,8 +73,7 @@ THE SOFTWARE. */
       this.iframe.setAttribute('allowFullScreen', '0');
 
       var divWrapper = document.createElement('div');
-      divWrapper.setAttribute('style', 'margin:0 auto;padding-bottom:56.25%;width:100%;height:0;position:relative;overflow:hidden;');
-      divWrapper.setAttribute('class', 'vimeoFrame');
+      divWrapper.setAttribute('style', 'width:638px; height:358px; overflow:hidden; margin:0 auto;');
```
Comment (joeypla) [inline src/Vimeo.js]: Where are these heights and widths coming from? They seem to be giving me a vimeo video with a grey border around it (Not proper sizing).

File: src/Vimeo.js [hunk 2]
```diff
@@ -353,11 +354,11 @@ THE SOFTWARE. */
 
   function injectCss() {
     var css = // iframe blocker to catch mouse events
+              '.vjs-vimeo { overflow: hidden }' +
               '.vjs-vimeo .vjs-iframe-blocker { display: none; }' +
               '.vjs-vimeo.vjs-user-inactive .vjs-iframe-blocker { display: block; }' +
               '.vjs-vimeo .vjs-poster { background-size: cover; }' +
-              '.vjs-vimeo { height:100%; }' +
-              '.vimeoplayer { width:100%; height:180%; position:absolute; left:0; top:-40%; }';
+              '.vimeoplayer {display:block; width:638px; height:558px; margin:0 auto;margin-top:-150px;}';
```
Comment (joeypla) [inline src/Vimeo.js]: Same as above. Am I missing something?

**Comment by monstrfolk** [comment]
Do not understand the CSS changes either. Created a new fork without it. Works much better.

https://github.com/rjgb/videojs-vimeo

**Comment by hendrathings** [comment]
will create new pull request

**Pull request closed without merging**